cloud_functions/flagCryptominer.js

```diff
@@ -0,0 +1,35 @@
+const utils = require('./utils')
+const { ValidationError }  = require('./ValidationError')
+
+
+const run = (mixpanel, message) => {
+  try {
+    const userSubjectId = validate(message)
+    if (userSubjectId) {
+      mixpanel.people.set(`google:${userSubjectId}`, { 'isCryptominer': true }, err => { if (err) throw err })
+    }
+  } catch (error) {
```

Comment:
TOL: Another way to factor this would be to move the `try`/`catch` into a wrapper, in order to fully separate out all the bits related to special error handling (which might make it easier to document). E.g.
```
const run = withErrorLogging((mixpanel, message) => {
  ...
})
```

Reply:
Ah, of course. Just like we do in terra-ui. I'll probably make that change. Thanks!